src/dex-ui/store/poolsSlice/poolsSlice.ts

```diff
@@ -201,6 +207,111 @@ const createPoolsSlice: PoolsSlice = (set, get): PoolsStore => {
       }
       app.setFeaturesAsLoaded(["addLiquidityTransactionState"]);
     },
+    sendCreatePoolTransaction: async (params: SendCreatePoolTransactionParams) => {
+      const { wallet, app, pools } = get();
+      const { network } = get().context;
+      app.setFeaturesAsLoading(["createPoolTransactionState"]);
+      const firstTokenAddress = TokenId.fromString(params.firstToken.address).toSolidityAddress();
+      const secondTokenAddress = TokenId.fromString(params.secondToken.address).toSolidityAddress();
+      const firstTokenQuantity = isHbarToken(params.firstToken.address)
+        ? BigNumber(0)
+        : wallet.getTokenAmountWithPrecision(params.firstToken.address, params.firstToken.qunatity);
+      const secondTokenQuantity = isHbarToken(params.secondToken.address)
+        ? BigNumber(0)
+        : wallet.getTokenAmountWithPrecision(params.secondToken.address, params.secondToken.qunatity);
+
+      const signingAccount = wallet.savedPairingData?.accountIds[0] ?? "";
+      const walletAddress = AccountId.fromString(signingAccount).toSolidityAddress();
+      const provider = DexService.getProvider(network, wallet.topicID, signingAccount);
+      const signer = DexService.getSigner(provider);
+
+      const tokenAHbarQty = isHbarToken(params.firstToken.address) ? params.firstToken.qunatity : 0.0;
+      const tokenBHbarQty = isHbarToken(params.secondToken.address) ? params.secondToken.qunatity : 0.0;
+      const HbarAmount = tokenAHbarQty + tokenBHbarQty;
+      const createPoolContractAddress = ContractId.fromString("");
```

Comment:
I have kept this filed as empty for now to quickly incorporate the new changes, I will update this on Monday.

Reply:
Done